app/server/src/errors/handleError.ts

```diff
@@ -0,0 +1,31 @@
+import { Request, Response } from "express";
+import { uid } from "uid";
+import { reqWithError } from "../logging";
+import { BeebopError } from "./beebopError";
+
+function sendError(response, error, status = 500) {
```

Comment:
`handleError` should be the only place where we write error responses, so moved `sendError` to be internal to this module.